src/ios/CDVWKInAppBrowser.m

```diff
@@ -257,16 +257,18 @@ - (void)openInInAppBrowser:(NSURL*)url withOptions:(NSString*)options
     
     [self.inAppBrowserViewController navigateTo:url];
     if (!browserOptions.hidden) {
-        [self show:nil withNoAnimate:browserOptions.hidden];
+        [self show:nil withNoAnimate:browserOptions.hidden displayCustomToolbar:browserOptions.customToolbar toolbarTitle:browserOptions.customToolbarTitle toolbarForegroundColor:[CDVWKInAppBrowserViewController colorFromHexString:browserOptions.customToolbarFg] toolbarBackgroundColor:[CDVWKInAppBrowserViewController colorFromHexString:browserOptions.customToolbarBg]];
```

Comment:
This line is so long. What do you think of breaking it into multiple lines, so that it's easier to read?